borg/testsuite/helpers.py

```diff
@@ -236,6 +237,45 @@ def test_patterns_fnmatch(pattern, expected):
 
 @pytest.mark.parametrize("pattern, expected", [
     # "None" means all files, i.e. all match the given pattern
+    ("*", None),
+    ("**/*", None),
+    ("/**/*", None),
+    ("/./*", None),
+    ("*/*", None),
+    ("*///*", None),
+    ("/home/u", []),
+    ("/home/*",
```

Comment:
hmm, why does this match /home/user/.profile?
we did specify that star does not match a path separator.